transaction-status/src/parse_instruction.rs

```diff
@@ -124,7 +123,7 @@ pub fn parse(
         ParsableProgram::Vote => serde_json::to_value(parse_vote(instruction, account_keys)?)?,
     };
     Ok(ParsedInstruction {
-        program: format!("{:?}", program_name).to_kebab_case(),
+        program: *program_name,
```

Comment:
Hmm, I think this will be a breaking change because `ParseableProgram` is serialized in json as camelCase, instead of the current kebab case